markov.js

```diff
@@ -1,4 +1,36 @@
-function buildChains(corpus, chainLength) {
+const fixCapitalisation = sentences => {
+    for (let i = 0; i < sentences.length; i++) {
+        sentences[i][0] = sentences[i][0].charAt(0).toUpperCase() + sentences[i][0].slice(1);
+        for (let j = 0; j < sentences[i].length; j++) {
```

Comment:
I mean, if you really want. It seems kind of excessive. A better approach might be leaving apostrophes in when we make chains? We probably can't get away from capitalizing I/I'll if it's important to you but we're going to lose proper nouns anyway.